docs: Clarify ingress gateway's -address flag (#10810)
hashicorp/consul: website/content/commands/connect/envoy.mdx

  `-register`. This takes the form of `<ip address>:<port>` but also supports go-sockaddr
  templates.

>   For terminating and mesh gateways, this is the IP and port on which
>   to receive traffic from services in the mesh. For ingress gateways, this is
>   the IP and port at which to instantiate a `/ready` HTTP endpoint which is used
>   by Consul for health checking the gateway. The specified IP will also be used

Reviewer: ```suggestion
  If Envoy is configured as a terminating or mesh gateway, traffic from services within the mesh will be received at the specified IP and port. 
  
  If Envoy is configured as an ingress gateway, a `/ready` HTTP endpoint will be instantiated at the specified IP and port. Consul uses `/ready` HTTP endpoints to check gateway health. The specified IP will also be used
```
We generally avoid passive voice, but in this instance I think using PV improves readability. Your original text was grammatically sound, so feel free to ignore this suggestion if you disagree.

  by the ingress gateway when instantiating user-defined listeners
  configured in the
  [ingress gateway](/docs/connect/gateways/ingress-gateway) configuration entry.

>   ~> **Note**: Ensure that user-defined ingress gateway listeners utilize a

Reviewer: ```suggestion
  ~> **Note**: Ensure that user-defined ingress gateway listeners use a
```
Nothing wrong with "utilize"; "use" is just shorter.

     different port than the port specified in `-address` so that they do not
     conflict with the health check endpoint.

- `-admin-access-log-path` The path to write the access log for the administration
  server. If no access log is desired specify `/dev/null`. By default it will
  use `/dev/null`.

- `-bind-address` - The bind address to use instead of the default binding rules.

>   Speciifed as `<name>=<ip>:<port>` pairs. This flag may be specified multiple times

Reviewer: ```suggestion
  Specified as `<name>=<ip>:<port>` pairs. This flag may be used multiple times
```
